router/auth.js

```diff
@@ -5,17 +5,19 @@ const router = express.Router()
 
 router.post('/register',async(req,res)=>{
 
-   const{name,email,phoneno,password}=req.body;
-   if(!name||!email||!phoneno||!password){
+   const{name,email,createpassword,confirmpassword}=req.body;
+   if(!name||!email||!createpassword||!confirmpassword){
```

Comment:
There is not need to check for confirmpassword since the password is confirmed in the front-end if they match or not.
```suggestion
   if(!name || !email || !password){
```